### DISCUSSION TITLE
Resource Loading/Transition Animation

### DISCUSSION THREAD

**Discussion by mahfoudfx** [filamentphp/filament#3870] [Ideas]
Hello,
On some slow servers or when loading/switch some resources with big data, in addition to livewire itself, it may take a few moment to load. Awaiting for the new resource to load, Filament hang on the old screen which gives a feeling of slowness and lake of responsiveness while an animation could give more smooth feeling. Same case for the Modals.
A noticed that the "Create" button trigger an animation of the button only when calling a Modal, while it doesn't when sending to the create page.
If it is not already implemented, my suggestion is to unify the animations triggered after each action on Filament (Buttons, Resource loading), which will allow a UI consistency, customization and code maintenance.

**Comment by dangnhdev** [comment]
Did you found anyway to do this? I'm working with Tesseract.js library, first time load is very long and I want to show init progress to user asynchronously.
